scripts/release.sh

```diff
@@ -0,0 +1,31 @@
+# /bin/bash
+
+set -e
+
+VERSION=${1}
+
+if [ -z "${VERSION}" ]; then
+    echo "usage: $0 <version>"
+    exit 1
+fi
+
+rm -rf build/
+mkdir build/
+cd build/
+
+# Copy the WASM binary
+cp ../reproducible-builds/wasm-client.wasm lnc-v${VERSION}-alpha.wasm
+
+# Zip up the iOS and Android binaries
+pushd ~/gocode/src/github.com/lightninglabs/lightning-node-connect/build/
+zip -r -X lnc-v${VERSION}-alpha-ios.zip ios/
+zip -r -X lnc-v${VERSION}-alpha-android.zip android/
+popd
+
+# Copy the iOS and Android binaries
+cp ${GOPATH}/src/github.com/lightninglabs/lightning-node-connect/build/*.zip .
```

Comment:
Same here re static path. Since projects don't need to be below the `GOPATH` directory anymore, this should also just be `../build/*.zip .`.

If we want to make the script a bit more portable, this is something I use in my scripts to reliably get the current directory where the **script** resides:

```
DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
``

Reply:
Nice 👌. I've updated the script to use this and it works great. Thanks